Example/Tests/Message.swift

```diff
@@ -9,6 +9,44 @@
 import Foundation
 import CoreData
 import DataStoreKit
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```

Comment:
this was automatically added when the code was converted to swift 3 using xcode converter